src/cljs/main/broadfcui/page/workspace/details.cljs

```diff
@@ -129,10 +131,26 @@
         [:div {:style {:minHeight "0.5rem"}}
          [ProtectedBanner (select-keys @state [:workspace :workspace-error])]
          [BucketBanner (select-keys @state [:bucket-access? :bucket-status-code])]]
-        [:div {:style {:marginTop "2rem" :paddingLeft "1.5rem" :fontSize "125%"}}
-         "Workspace: "
-         [:span {:style {:fontWeight 500}}
-          (:namespace workspace-id) "/" (:name workspace-id)]]
+        [:div {:style {:marginTop "1rem" :padding "0 1.5rem"
+                       :display "flex" :justifyContent "space-between"}}
+         [:div {:style {:fontSize "125%"}}
+          "Workspace: "
+          [:span {:style {:fontWeight 500}} (:namespace workspace-id) "/" (:name workspace-id)]]
+         [:div {:style {:marginRight "-0.5rem"}}
```

Comment:
Was there not a way to apply this margin change further down? I'd think to put it on the style for the `FoundationTooltip` below.